ang/civicase/ActivityFeed.js

```diff
@@ -431,28 +424,49 @@
        * Sets Activity Details Panel affix offsets
        */
       function affixActivityDetailsPanel () {
-        $timeout(function () {
-          $activityDetailsPanel = $element.find('.panel');
-          $filter = $('.civicase__activity-filter');
-          $feedListContainer = $('.civicase__activity-feed__list-container');
-          $tabs = $('.civicase__dashboard').length > 0 ? $('.civicase__dashboard__tab-container ul.nav') : $('.civicase__case-body_tab');
-          $toolbarDrawer = $('#toolbar');
-
-          $activityDetailsPanel.affix({
-            offset: {
-              top: $element.find('.panel').offset().top - ($toolbarDrawer.height() + $tabs.height() + $filter.height()),
-              bottom: $($document).height() - ($feedListContainer.offset().top + $feedListContainer.height())
-            }
-          }).on('affixed.bs.affix', function () {
-            $activityDetailsPanel
-              .css('top', ($toolbarDrawer.height() + $tabs.height() + $filter.height()))
-              .css('padding-top', 32);
-          }).on('affixed-top.bs.affix', function () {
-            $activityDetailsPanel
-              .css('top', 'auto')
-              .css('padding-top', 0);
-          });
+        $activityDetailsPanel = $element.find('.civicase__activity-panel');
+        $filter = $('.civicase__activity-filter');
+        $feedListContainer = $('.civicase__activity-feed__list-container');
+        $tabs = $('.civicase__dashboard').length > 0 ? $('.civicase__dashboard__tab-container ul.nav') : $('.civicase__case-body_tab');
+        $toolbarDrawer = $('#toolbar');
+
+        $activityDetailsPanel.affix({
+          offset: {
+            top: getTopOffset(),
+            bottom: getBottomOffset()
+          }
         });
+
+        $activityDetailsPanel.on('affixed.bs.affix', function () {
+          $activityDetailsPanel
+            .css('top', getDistanceFromTop())
+            .width($element.width());
```

Comment:
@reneolivo as i wrote in a previous comment, the value of  `width` doesn't need to be set via JS. All that we need is a class to toggle whenever `affixed.bs.affix` or `affixed-top.bs.affix` is emitted

Reply:
Removed this. The class already exists. It's `affix`, which is added automatically by the plugin.